atomisticparsers/gromacs/parser.py

```diff
@@ -83,27 +83,46 @@ def str_to_input_parameters(val_in):
                     parameters[val_array.group(1)].append(value[0] if len(value) == 1 else value)
             return parameters
 
-        def str_to_energies(val_in):
-            energy_keys_re = re.compile(r'(.+?)(?:  |\Z| P)')
-            keys = []
-            values = []
-            energies = dict()
-            for val in val_in.strip().split('\n'):
-                val = val.strip()
-                if val[0].isalpha():
-                    keys = [k.strip() for k in energy_keys_re.findall(val)]
-                    keys = ['P%s' % k if k.startswith('res') else k for k in keys if k]
+        def str_to_energies(energies_block):
+
+            max_char_name = 14
+            rows = energies_block.strip().split("\n")
+            names = [rows[i_row] for i_row in range(len(rows)) if i_row % 2 == 0]
```

Comment:
maybe you can do this in a single loop

Reply:
A normal for loop and then use append()?

Reply:
I think this becomes moot.

Reply:
assuming what I said above is correct, then done.